packages/expo-audio/src/Audio.types.ts

```diff
@@ -128,16 +202,31 @@ export enum IOSOutputFormat {
   ENHANCEDAC3 = 'ec-3',
 }
 
-// @docsMissing
+/**
+ * Audio quality levels for recording.
+ *
+ * Predefined quality levels that balance file size and audio fidelity.
+ * Higher quality levels produce better sound but larger files and require more processing power.
+ */
 export enum AudioQuality {
+  /** Minimum quality - smallest file size, lowest fidelity. */
```

Comment:
```suggestion
  /** Minimum quality: smallest file size, lowest fidelity. */
```

Let's prefer using colon or mdhash instead of endash.